db/seeds.rb

```diff
@@ -5,3 +5,52 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# users
+
+admin = [true, true, true, false, false, false, false, false, false, false, false, false, false, false, false]
+first_name = ["Peter", "Veronika", "Thomas", "Karl", "Morgan", "Tim", "Marita", "Bernhard", "Bianca", "Sia", "Claudio", "Laura", "Marie", "Luis", "Alex", "Manfred", "Alissa", "Ken", "Sebastian", "Zita"]
+last_name = ["Hill", "Landberg", "Hintermeyer", "Leoni", "Richardson", "Lagerfeld", "Arana", "Apmann", "Arco", "Sadi", "Tiedemann", "Wittelberg", "Neisemeier", "Iarossi", "Jenkara", "Scherhausen", "Linkhammer", "Ginser", "Patoli", "Dahme"]
+additional_infos = ["Hey there - your cardiologist is here.", "I am new to the team - whoop, whoop.", "Busy", "📈🩺👨🏼‍⚕️", "Saving some babies! #neonatal", "I want icecream 🍦",  "👩🏼‍⚕️", "In the gym", "Do not disturb - not taking any offers🤪", "Nihil since causa", "29.05.2022", "💙", "Passionate delivery nurse #lovemyjob", "All these christmas lights 🎶🎵", "tired😴", "Hey there!", "Pregnant - so excited🤰🏼", "Caught Covid 😥", "Looking forward to my vacantions." "I got a new pair of shoes😎"]
+experience_level = ["experienced", "experienced", "experienced", "beginner", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "experienced", "beginner", "beginner", "experienced", "beginner", "experienced", "experienced"]
+# functional_role
+names1 = ["nurse manager", "nurse manager", "nurse manager", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse", "labor & delivery nurse", "emergency-room nurse", "intensive-care-unit nurse", "radiology nurse"]
```

Comment:
why not use functional_role as variable name?

Reply:
because the name in our schema model is name

Reply:
ah okay. :) 
I think it would be easier to understand (especially later in the functions), if you would use the descriptive name. You can also get rid of the comment then.